src/addr.rs

```diff
@@ -439,7 +443,10 @@ impl PhysAddr {
     /// ## Panics
     ///
     /// This function panics if a bit in the range 52 to 64 is set.
+    /// If the `memory_encryption` feature is available and has been enabled, this function also
+    /// panics fails if the encryption bit is manually set in the address.
```

Comment:
```suggestion
    /// If the `memory_encryption` feature has been enabled and an encryption bit has been configured, this function
    /// panics if the encryption bit is set in the address.
```

Ditto for `try_new`.